modyn/tests/trainer_server/internal/trainer/remote_downsamplers/test_abstract_remote_downsampling_strategy.py

```diff
@@ -0,0 +1,17 @@
+# pylint: disable=abstract-class-instantiated
+from unittest.mock import patch
+
+from modyn.trainer_server.internal.trainer.remote_downsamplers.abstract_remote_downsample_strategy import (
+    AbstractRemoteDownsamplingStrategy,
+)
+
```

Comment:
Ok we definitely need more tests here. This file contains functions with logic, such as `get_tensors_subset` and `sample` and both of them are not tested here right now

Reply:
I've added some tests for `get_tensors_subset`. Regarding `sample `, it is tested in the actual implementations (GradNorm, Loss ..)

Reply:
Mmmm I think you are looking at the old version of `AbstractRemoteDownsamplingStrategy`. In this PR, it has just abstract classes. `Sample` and `get_probabilities` were in the previous version

```
@abstractmethod
def init_downsampler(self) -> None:
  raise NotImplementedError

@abstractmethod
def inform_samples(self, sample_ids: list[int], forward_output: torch.Tensor, target: torch.Tensor) -> None:
  raise NotImplementedError

@abstractmethod
def select_points(self) -> tuple[list[int], torch.Tensor]:
  raise NotImplementedError
```

Reply:
Oh - sorry, this was what Github has shown me when clicking view file.